README.md

```diff
@@ -1 +1,47 @@
-# Hello-Microverse-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is the first html and css project of the first block in which linters are integrated.
+> This project is part of my Microverse exercises, where we learn how to set a up a new repository in a professional manners.
+> This project doesn't require any complex coding. The goal here is to master all of the tools and best practices that are used in a professional development environment.
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+This is project is very simple project and it has no any further testing and deployment stages.
+
+### Prerequisites
+
+simple knowledge of HTML, CSS and git/Github is required
+
+### Setup
+
+Any code editor can be used, although in this instance, VS code is used.
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/KenbonTN)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kenbon-teshome/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+I am greateful for microvese community!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
#### Great work implementing this project. I know you put in a lot of work to get it through. Kudos 👏 💪 

- [ ]  I can see that you used the former Microverse template which is great but based on the requirements. Microverse has a new recommended template that students use for all their projects in Microverse. Kindly use this template to create a professional README.md file https://github.com/microverseinc/readme-template. For this template attached, kindly do the following when customizing it to suit this project:
- Remove the Microverse logo. If you have your logo, you can use it but I would advise you to remove it. 🙏 
- For the **_Tech Stack_** Section, Kindly remove the _**Server**_ and _**Database**_ subsections because the technologies listed there are not relevant to this project. For the _**Client**_ section, kindly include `HTML` and `CSS` 👍
- For the **_Key Features_** section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file. 👍 
- For the **_Prerequisite_** Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:
  - Installation of `Git`
  - Installation of `NPM`
  - A working browser
  - Internet connection
  - A code editor like `Visual Studio Code`
  - Installation of `Live Server` plugging in Visual Studio Code
  - Any other tool you can think about
- For the _**Setup**_ section, kindly add instructions for cloning your repository on a local machine 👍 
- For the _**Install**_ section, kindly add instructions for setting up linters used in this project
-  For the _**Usage**_ section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.
- For the _**MIT**_ section, kindly add a valid LICENSE. You can follow the instructions on the right side of the article attached below to set up a valid LICENSE
https://choosealicense.com/licenses/mit/
#### Kindly implement all the changes listed above. It will make your _**README.md**_ file look professional and it will give you an idea of how to set _**README.md**_ files for most of the projects you will execute as a Microverse student. 👍 💻